cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -384,6 +385,29 @@ class LeadImpl extends ServerImpl with Lead with Logging {
     finalConf
   }
 
+  def getDynamicOverrides: Properties = {
+    val dynamicOverrides = new Properties()
+
+    def replace(key: String, value: String, newValue: String) = {
+      assert (value.indexOf("/tmp") >= 0)
+      dynamicOverrides.setProperty(key, value.replace("/tmp", newValue))
+    }
+
+    val workingDir = System.getProperty(
+      com.pivotal.gemfirexd.internal.iapi.reference.Property.SYSTEM_HOME_PROPERTY, ".")
+    val defaultConf = ConfigFactory.parseResources("jobserver-defaults.conf")
+
+    var key = "spark.jobserver.filedao.rootdir"
+    replace(key, defaultConf.getString(key), workingDir)
```

Comment:
Instead of hackish replace of "/tmp" can't we have a token say "${WORKING_HOME}" in jobserver-defaults.conf that gets substituted by the value of environment variable which we set appropriately?